Apps/Common/SIMCoupled.h

```diff
@@ -53,7 +53,13 @@ template<class T1, class T2> class SIMCoupled
   }
 
   //! \brief Computes the solution for the current time step.
-  virtual bool solveStep(TimeStep& tp, bool firstS1 = true)
+  virtual bool solveStep(TimeStep& tp)
+  {
+    return this->solveStep(tp, true);
+  }
+
+  //! \brief Computes the solution for the current time step.
+  virtual bool solveStep(TimeStep& tp, bool firstS1)
```

Comment:
Sorry, but I don't see what was the problem here. The solveStep method is only here and in the sub-class SIMCoupledSI. Why can't they have the same signature? Other classes withe the solveStep method does not belong to this hierarchy, so that shouldn't matter There are many examples in the IFEM library with default values on arguments in virtual methods with no need to make a duplicate like in this case. So please explain better for a novice why this is wrong (so I don't do it again;-)